examples/apps/heart_disease_eval/client/heart_gui.py

```diff
@@ -0,0 +1,495 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import random
+import json
+import argparse
+import logging
+import base64
+
+import tkinter as tk
+import tkinter.messagebox as messagebox
+import tkinter.font as font
+
+from service_client.generic import GenericServiceClient
+import utility.utility as utility
+import worker.worker_details as worker
+import json_rpc_request.json_rpc_request as jrpc_request
+from connectors.direct.direct_adaptor_factory_wrapper \
+	import DirectAdaptorFactoryWrapper
+import config.config as pconfig
+import utility.logger as plogger
+
+# Remove duplicate loggers
+for handler in logging.root.handlers[:]:
+    logging.root.removeHandler(handler)
+logger = logging.getLogger(__name__)
+TCFHOME = os.environ.get("TCF_HOME", "../../")
+
+# -----------------------------------------------------------------
+# Validates that input is a non-negative int
+def intValidate(text):
+	if str.isdigit(text) or text == "":
+		return True
+	else:
+		return False
+
+# Validates that input is a non-negative, non-special float
+def floatValidate(text):
+	if text == "":
+		return True
+	try:
+		float(text)
+		if float(text) < 0.0 or float(text) == float("NaN") \
+			or float(text) == float("INF") or float(text) == float("-INF"):
+			return False
+		return True
+	except ValueError:
+		return False
+
+# User entry for ints
+class intEntry:
+	def __init__(self, master, name):
+		global cur_row
+		label = tk.Label(master, text=name)
+		label.grid(row=cur_row, column=0, sticky="e", pady=(5,0))
+		validate = (master.register(intValidate))
+		self.entry = tk.Entry(master, validate="all", 
+			validatecommand=(validate, "%P"))
+		self.entry.grid(row=cur_row, column=1, padx=(10,0), pady=(5,0),
+			sticky="w")
+		cur_row += 1
+
+	def get(self):
+		# Fails if empty field
+		try:
+			return int(self.entry.get())
+		except ValueError:
+			return None
+
+	def enable(self):
+		self.entry.config(state=tk.NORMAL)
+
+	def disable(self):
+		self.entry.config(state=tk.DISABLED)
+
+# User entry for floats
+class floatEntry:
+	def __init__(self, master, name):
```

Comment:
intEntry and floatEntry class looks similar. Is there any generic class and type we can use in Python ? of use IsInstance() function to compare the type